pulsar-client-cpp/wireshark/pulsarDissector.cc

```diff
@@ -157,10 +232,7 @@ struct RequestData {
     uint32_t ackFrame;
     nstime_t ackTimestamp;
 
-    RequestData()
-            : requestFrame(UINT32_MAX),
-              ackFrame(UINT32_MAX) {
-    }
+    RequestData(): requestFrame(UINT32_MAX),ackFrame(UINT32_MAX) {}
```

Comment:
In addition, I see many changes about code format, like here. The change is really redundant. For example, when I use `clang-format` to reformat your file again, this line also doesn't meet the code format.

```diff
-    RequestData(): requestFrame(UINT32_MAX),ackFrame(UINT32_MAX) {}
+    RequestData() : requestFrame(UINT32_MAX), ackFrame(UINT32_MAX) {}
```

Reply:
Please just retain the necessary changes. Regarding to the code style and format, we can use another PR to fix it.

Reply:
Make sense. I'll redo this change.